### ISSUE TITLE
[Bug]: Old icons improper appearance. 

### ISSUE THREAD

**Issue by Sniperexe** [sapondanaisriwan/AdashimaaTube#107]
### Describe the bug

Old icons do not appear properly on home page as well as under a video. See screenshots below.
(Issues came along with the new version of this extension; everything was fine in 2.2.6)

### Screenshots

![00](https://github.com/sapondanaisriwan/AdashimaaTube/assets/97830738/3da4434a-479e-4a32-a239-7c6470d23e19)
![99](https://github.com/sapondanaisriwan/AdashimaaTube/assets/97830738/193a9f27-64af-4cdf-9530-55feca175870)

### What browser are you using?

Latest Version of Chrome

### What version of AdashimaaTube are you using?

2.2.7

**Comment by sapondanaisriwan** [comment]
Why does YouTube have too many svg paths? On my end, everything works perfectly.
![msedge_FIgfcgY2ul](https://github.com/sapondanaisriwan/AdashimaaTube/assets/64634605/e7006475-af1e-4213-9fbe-197369845d5f)

**Comment by sapondanaisriwan** [comment]
Please use the previous version until I figure out how to fix it.

**Comment by Sniperexe** [comment]
Yup that's what I did

**Comment by Sniperexe** [comment]
Okay so youtube has been really fucking around with us as many occurrences keep emerging. Keep in mind the option "Old Icons" is enabled.


A) In this section I will talk about the version 2.2.6

Apparently having different accounts plays a major role on the way youtube's UI is displayed to you.
Using my main account everything seems to appear just fine, whereas on my second which I barely use, almost every icon appears with a different shape rather than the "old" one. See screenshot below.

![hhh](https://github.com/sapondanaisriwan/AdashimaaTube/assets/97830738/96ceb064-74a8-4949-ac82-ca6f484d9f94)


B) The following text is in regards of version 2.2.7

Unexpecting things take place in this version as certain icons appear improperly on my main account instead on my second. On my latter they seem to be fine this time. The screenshot is alike to which this ticket was created initially for, but I'm writing this whole thing to provide you with a clearer version of this issue.

![ggg](https://github.com/sapondanaisriwan/AdashimaaTube/assets/97830738/87d88052-ae83-4638-b4db-df9f56a8298a)


Last but not least, I noticed that the "save" button was embedded in the three dots where the report button is, but was unable to capture it. This particular change arose twice randomly, so I haven't been able to recreate this in order to capture it. The version I was then using is 2.2.6 but can't assure you it won't happen in 2.2.7. For the time being 2.2.6 appears to be the most stable for me and consequently that's what I'm using.  

I hope I didn't bore you, take as long as you need to fix these.

**Comment by Sniperexe** [comment]
> > Okay so youtube has been really fucking around with us as many occurrences keep emerging. Keep in mind the option "Old Icons" is enabled.
> > A) In this section I will talk about the version 2.2.6
> > Apparently having different accounts plays a major role on the way youtube's UI is displayed to you. Using my main account everything seems to appear just fine, whereas on my second which I barely use, almost every icon appears with a different shape rather than the "old" one. See screenshot below.
> > ![hhh](https://user-images.githubusercontent.com/97830738/239378888-96ceb064-74a8-4949-ac82-ca6f484d9f94.PNG)
> > B) The following text is in regards of version 2.2.7
> > Unexpecting things take place in this version as certain icons appear improperly on my main account instead on my second. On my latter they seem to be fine this time. The screenshot is alike to which this ticket was created initially for, but I'm writing this whole thing to provide you with a clearer version of this issue.
> > ![ggg](https://user-images.githubusercontent.com/97830738/239388817-87d88052-ae83-4638-b4db-df9f56a8298a.PNG)
> > Last but not least, I noticed that the "save" button was embedded in the three dots where the report button is, but was unable to capture it. This particular change arose twice randomly, so I haven't been able to recreate this in order to capture it. The version I was then using is 2.2.6 but can't assure you it won't happen in 2.2.7. For the time being 2.2.6 appears to be the most stable for me and consequently that's what I'm using.
> > I hope I didn't bore you, take as long as you need to fix these.
> 
> @Sniperexe In my opinion, it should be fixed in v2.2.8. ![image](https://user-images.githubusercontent.com/64634605/239933333-f7a58a1a-d15e-4997-a52c-a10013e82970.png)

yup, everything's fixed. Thanks.
there's a small detail though that does not really concern me at all but would like to let you know that in my second account when liking a comment the collor is set to black; under the videos it's normal. In my main account eveything operates fine, but since my second encountered "issues" in a past basis, I'm letting you know. I guess my latter account is somehow receiving earlier or different updates regarding youtube's UI as it always has been displaying the elements kinda differently before these updates came to my main one over time. The reason I'm mentioning this is that there are people likely who come across these "early" changes in contrast to my main account. Personally doesn't affect the particular change at all because I use my main account, thanks again.

**Comment by Sniperexe** [comment]
That's working, apologies for late reply

**Comment by Sniperexe** [comment]
"Last but not least, I noticed that the "save" button was embedded in the three dots where the report button is, but was unable to capture it. This particular change arose twice randomly, so I haven't been able to recreate this in order to capture it." 
I did capture it this time but I don't think it's sth to worry about (at least for the time being) but I'm posting it here in case you wanna know what's it about. Goes away with just a refresh.

![HGKFYGK](https://github.com/sapondanaisriwan/AdashimaaTube/assets/97830738/81f92a5e-36a4-4106-89d5-7aaa793ef28e)
